### PR TITLE
Update install-and-update.md

### PR DESCRIPTION
**Pull request by sedgwickz**
wrangler cli latest 'version' param seems not work

### CODE REVIEW

**Comment by deadlypants1973** [comment]
Screenshot 2023-11-29 at 11 53 56


hi @sedgwickz , all three are working for me. what error are you getting?

**Comment by sedgwickz** [comment]
> Screenshot 2023-11-29 at 11 53 56
> hi @sedgwickz , all three are working for me. what error are you getting?

Hi @deadlypants1973, I'm not sure why it happened. But compared with yours I found my version was out. I've upgraded it and it works well. Thanks.

image

File: content/workers/wrangler/install-and-update.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To check your Wrangler version, run:
 ```sh
 $ npx wrangler --version
 // or run:
-$ npx wrangler version
+$ npx wrangler -V
```
Comment (deadlypants1973) [inline content/workers/wrangler/install-and-update.md:32]:
```suggestion
$ npx wrangler version
// or
$ npx wrangler -V
```

File: content/workers/wrangler/install-and-update.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ To check your Wrangler version, run:
 ```sh
 $ npx wrangler --version
 // or run:
```
Comment (deadlypants1973) [inline content/workers/wrangler/install-and-update.md]:
```suggestion
// or
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**